make/kube

```diff
@@ -7,22 +7,7 @@ cd "${GIT_ROOT}"
 source .envrc
 source ${GIT_ROOT}/make/include/versioning
 
-if [ "${USE_SLE_BASE}" = "false" ]; then
-    FISSILE_DEFAULTS_FILE="bin/settings/opensuse42.env"
-else
-    FISSILE_DEFAULTS_FILE="bin/settings/sle12.env"
-fi
-
-DOMAIN=${DOMAIN:-}
-
-NETWORK_ENV=bin/settings/network.env
-if test -n "${DOMAIN}"; then
-    TMP=$(mktemp -d)
-    cp "${NETWORK_ENV}" "${TMP}"
-    trap "rm -rf ${TMP}" EXIT
-    NETWORK_ENV="${TMP}/$(basename "${NETWORK_ENV}")"
-    sed -e "s/\\bcf-dev.io/${DOMAIN}/g" -i "${NETWORK_ENV}"
-fi
+# Note: sle vs opensuse is decided in .envrc
```

Comment:
Not really, or at least not anymore. It is decided in the `bin/fissile` wrapper; `.envrc` just makes sure the `USE_SLE` variables exists with a default value.

Reply:
will fix.

Reply:
Done with commit bbc691be8f3efd1549a83082a0c74294a99f487d